pkg/sql/importer/import_stmt_test.go

```diff
@@ -3896,6 +3896,8 @@ func benchUserUpload(b *testing.B, uploadBaseURI string) {
 			`IMPORT INTO t CSV DATA ('%s%s')`,
 			uploadBaseURI, testFileBase,
 		))
+
+	b.StopTimer()
```

Comment:
I'm skeptical of this benchmark. On my M1 Mac it now takes 0.12ns/op. But it looks like it was similarly fast before. The benchmark comparison sheet [recorded it at 0.49ns/op](https://docs.google.com/spreadsheets/d/15upfFc-6YQjWkp4OqHqwFkh2SY8344bnEZbOYYYHGvQ/edit#gid=1&range=B2606) before the regression.